js/src/views/Accounts/List/list.js

```diff
@@ -80,20 +81,31 @@ class List extends Component {
       return (
         <div
           className={ styles.item }
-          key={ address }>
-          <Summary
-            link={ link }
-            account={ account }
-            balance={ balance }
-            owners={ owners }
-            handleAddSearchToken={ handleAddSearchToken }
-            showCertifications
-          />
+          key={ address }
+        >
+          { this.renderSummary(account, balance, owners) }
         </div>
       );
     });
   }
 
+  renderSummary (account, balance, owners) {
+    const { handleAddSearchToken, link, summary } = this.props;
+
+    const props = {
+      account, balance, handleAddSearchToken, link, owners,
+      showCertifications: true
+    };
+
+    if (summary) {
+      return createReactElement(summary, props);
```

Comment:
I'm really not convinced we should be going down this route with createElement, even if it is in a single place. Now there is magic is the caller, instead of being explicit and here we are doign what JSX should be doing.